### DISCUSSION TITLE
problems with rules

### DISCUSSION THREAD

**Discussion by bookvik** [PostgREST/postgrest#1792] [Q&A] [answered]
due to the fact that the code for any mutation substitutes 1, it becomes impossible to prohibit deletion at the rule level

![image](https://user-images.githubusercontent.com/204606/113721769-7b529e80-96f8-11eb-8254-6a8f6df6ac08.png)

![image](https://user-images.githubusercontent.com/204606/113721842-8a395100-96f8-11eb-8f42-4cde5144d094.png)

![image](https://user-images.githubusercontent.com/204606/113721922-99b89a00-96f8-11eb-9004-d6eca7d77274.png)

![image](https://user-images.githubusercontent.com/204606/113722060-b9e85900-96f8-11eb-91c1-eff3309b32fb.png)

how can this be fixed?

**Comment by bookvik** [comment, marked as answer]
haha i fixed at returning *

**Comment by wolfgangwalther** [comment]
It would be very helpful to not post pictures, but actual text. Text can be searched, copied, resized, whatever.

**Reply by bookvik** [reply]
sorry, ok
